src/project-manager.ts

```diff
@@ -852,10 +864,39 @@ export class ProjectConfiguration {
 			this.logger
 		);
 		this.service = ts.createLanguageService(this.host, this.documentRegistry);
+		const pluginLoader = new PluginLoader(this.rootFilePath, this.fs, this.initializationOptions, this.logger);
+		pluginLoader.loadPlugins(options, this.enableProxy.bind(this));
 		this.initialized = true;
 	}
 
 	/**
+	 * Replaces the LanguageService with an instance wrapped by the plugin
+	 * @param pluginModuleFactory function to create the module
+	 * @param configEntry extra settings from tsconfig to pass to the plugin module
+	 */
+	private enableProxy(pluginModuleFactory: PluginModuleFactory, configEntry: ts.PluginImport) {
```

Comment:
can this be called multiple times to wrap with multiple plugins?

Reply:
Yes, every time enableProxy is called, the current `this.service` is wrapped, so the desired result is `plugin2Wrapper(plugin2Wrapper(tsService))`

Reply:
A name like `wrapService` would make it more clearer to me then, `enable` always sounds like an idempotent function (like setting a boolean to `true`)